content/en/network_monitoring/performance/setup.md

```diff
@@ -182,7 +182,8 @@ To enable Network Performance Monitoring for Windows hosts:
 
 1. Install the [Datadog Agent][1] (version 7.27.1 or above) with the network driver component enabled.
 
-   During installation pass `ADDLOCAL="MainApplication,NPM"` to the `msiexec` command, or select "Network Performance Monitoring" when running the agent installation through the GUI.
+   During installation select the check box to consent to non-open source software.
+   During installation pass `ADDLOCAL="MainApplication,NPM"` to the `msiexec` command, or select "Network Performance Monitoring" when running the agent installation through the GUI. _(version 7.44 or below)_
```

Comment:
```suggestion
  _(version 7.44 or below)_ During installation pass `ADDLOCAL="MainApplication,NPM"` to the `msiexec` command, or select "Network Performance Monitoring" when running the agent installation through the GUI. 
   _(version 7.45 or above)_ During installation select the check box to consent to non-open source software. 
```

Reply:
That's a good call. I applied the suggestion, and I put the 7.45 instructions first because that's the latest / the version we encourage customers to use.